Drop-In/src/main/java/com/braintreepayments/api/DropInEvent.java

```diff
@@ -0,0 +1,116 @@
+package com.braintreepayments.api;
+
+import android.os.Bundle;
+import android.os.Parcel;
+import android.os.Parcelable;
+
+class DropInEvent implements Parcelable {
+
+    private final DropInEventType type;
+    private final Bundle payload;
+
+    static DropInEvent createSendAnalytics(String eventName) {
+        DropInEvent event = new DropInEvent(DropInEventType.SEND_ANALYTICS);
+        event.putString(DropInEventProperty.ANALYTICS_EVENT_NAME, eventName);
+        return event;
+    }
+
+    static DropInEvent createAddCardSubmitEvent(String cardNumber) {
+        DropInEvent event = new DropInEvent(DropInEventType.ADD_CARD_SUBMIT);
+        event.putString(DropInEventProperty.CARD_NUMBER, cardNumber);
+        return event;
+    }
+
+    static DropInEvent createCardDetailsSubmitEvent(Card card) {
+        DropInEvent event = new DropInEvent(DropInEventType.CARD_DETAILS_SUBMIT);
+        event.putParcelable(DropInEventProperty.CARD_DETAILS, card);
```

Comment:
Maybe we could just call this `CARD` instead of `CARD_DETAILS` to make it more clear that it's a card object being stored?

Reply:
Yeah I'm up for clarifying. I wasn't super sure on the names tbh.